contrib/linux/actions/scp.yaml

```diff
@@ -13,10 +13,6 @@
             type: 'string'
             description: 'Destination of files/directories'
```

Comment:
@nmaludy Can you please look into this change and the one mentioned in changelog so we can get it merged once that is resolved?

Thanks :)